src/components/MenuItemWithExternalIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import { ExternalLinkIcon } from "@chakra-ui/icons";
+import { Flex, MenuItem, MenuItemProps, Text } from "@chakra-ui/react";
+import React from "react";
+
+import { HSpacer } from "@/components/Spacer";
+
+type CustomProps = {
+  label: string;
+};
+
+type Props = CustomProps & MenuItemProps;
+
+const MenuItemWithExternalIcon = ({ label, ...menuItemProps }: Props) => {
```

Comment:
アイコン名に合わせて
```suggestion
const MenuItemWithExternalLinkIcon = ({ label, ...menuItemProps }: Props) => {
```
の方がよさそう